http-server/build.gradle

```
@@ -1,22 +1,43 @@
 plugins {
     id 'application'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
-    id 'de.undercouch.download' version '3.4.3'
 }
 
 archivesBaseName = "$group-$name"
 description = 'TripleA DropWizard Http Server'
-mainClassName = '' // org.triplea.game.client.HeadedGameRunner'
-
+mainClassName = 'org.triplea.server.http.ServerApplication'
 ext {
     releasesDir = file("$buildDir/releases")
 }
 
+
+configurations {
+    testImplementation {
+        // database-rider brings in slf4j-simple as a transitive dependency
+        // DropWizard has logback baked in and cannot have multiple slf4j bindings.
+        exclude group: 'org.slf4j', module: 'slf4j-simple'
+    }
+}
+
 dependencies {
-    implementation 'io.dropwizard:dropwizard-core:1.3.11'
     implementation project(':java-extras')
     implementation project(':http-clients')
+
+    implementation 'io.dropwizard:dropwizard-core:1.3.12'
+    implementation 'io.dropwizard:dropwizard-jdbi3:1.3.12'
+    implementation 'org.jdbi:jdbi3-core:3.8.2'
+    implementation 'org.jdbi:jdbi3-sqlobject:3.8.2'
+
+    implementation 'javax.xml.bind:jaxb-api:2.3.1'
+    implementation 'com.sun.xml.bind:jaxb-impl:2.3.2'
+    implementation 'com.sun.xml.bind:jaxb-core:2.3.0'
+    implementation 'javax.activation:activation:1.1.1'
+
+    runtimeOnly "org.postgresql:postgresql:$postgresqlVersion"
+
     testImplementation project(':test-common')
+    testImplementation 'com.github.database-rider:rider-junit5:1.5.2'
+    testCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: '1.3.12'
```

For consistency I'd prefer the 'usual syntax'
```suggestion
    testCompile group: 'io.dropwizard:dropwizard-testing:1.3.12'
```

---

Not sure what the problem I ran into there originally, regardless I was able to use the shorthand syntax in the end without issue, updated in: 624229878